src/config/db.config.ts

```diff
@@ -0,0 +1,13 @@
+export const dbConfig = {
+    HOST: 'localhost',
+    USER: 'root',
+    PASSWORD: '123456',
+    DB: 'testdb',
```

Comment:
can we change the name to 'moirelocal' so that the name isn't as generic as testdb